sources/api/schnauzer/src/helpers.rs

```diff
@@ -1368,9 +1374,106 @@ enum OciSpecSection {
     Capabilities,
     ResourceLimits,
 }
-
 derive_fromstr_from_deserialize!(OciSpecSection);
 
+#[derive(Deserialize, Debug, Clone, Copy)]
+#[serde(rename_all = "kebab-case")]
+enum Runtime {
+    Docker,
+    Containerd,
+}
+derive_fromstr_from_deserialize!(Runtime);
```

Comment:
nit follow style seen elsewhere.

```suggestion
}

derive_fromstr_from_deserialize!(Runtime);
```